.gitlab/functional_test/system_probe.yml

```diff
@@ -143,5 +150,121 @@ kitchen_test_system_probe_windows_x64:
     - export WINDOWS_DDNPM_SHASUM=$(inv release.get-release-json-value "$RELEASE_VERSION_7::WINDOWS_DDNPM_SHASUM")
     - pushd $DD_AGENT_TESTING_DIR
     - tasks/kitchen_setup.sh
+    - !reference [ .system_probe_file_diff_check ]
+  script:
+    - tasks/run-test-kitchen.sh windows-sysprobe-test $AGENT_MAJOR_VERSION
+
+kitchen_test_system_probe_linux_x64_manual:
```

Comment:
I am not sure having manual versions of all the jobs is a good idea. We should look at making a rule that will fallback to manual if there are no relevant file changes.

Reply:
It looks like the `CI_JOB_MANUAL` environment variable will be `true` if run manually, so we could skip the file check if so.